pip/templates/deploy.py

```diff
@@ -29,53 +29,22 @@
 # noinspection PyUnresolvedReferences
 import twine.commands.upload
 
-parser = argparse.ArgumentParser()
-parser.add_argument('repo_type')
-args = parser.parse_args()
-
-repo_type_key = args.repo_type
-
-pip_repositories = {
-    'snapshot' : "{snapshot}",
-    'release' : "{release}"
-}
-
-pip_registry = pip_repositories[repo_type_key]
-
-pip_username, pip_password = (
-    os.getenv('DEPLOY_PIP_USERNAME'),
-    os.getenv('DEPLOY_PIP_PASSWORD'),
-)
-
-if not pip_username:
-    raise Exception(
-        'username should be passed via '
-        'DEPLOY_PIP_USERNAME env variable'
-    )
-
-if not pip_password:
-    raise Exception(
-        'password should be passed via '
-        '$DEPLOY_PIP_PASSWORD env variable'
-    )
-
 with open("{version_file}") as version_file:
     version = version_file.read().strip()
-
 new_package_file = None
-
+new_wheel_file = None
 try:
-    new_package_file = "{package_file}".replace(".tar.gz", "-{}.tar.gz".format(version))
+    os.mkdir("./dist")
+    new_package_file = "./dist/" + "{package_file}".replace(".tar.gz", "-{}.tar.gz".format(version))
+    new_wheel_file = "./dist/" + "{wheel_file}".replace(".whl", "-{}.whl".format(version))
+
     shutil.copy("{package_file}", new_package_file)
+    shutil.copy("{wheel_file}", new_wheel_file)
 
     twine.commands.upload.main([
-        new_package_file,
-        '-u',
-        pip_username,
-        '-p',
-        pip_password,
-        '--repository-url',
-        pip_registry
+        './dist/*',
+        '--repository',
+        'codeartifact'
```

Comment:
here's where i'm a bit leery of removing the previous `snapshot` & `release` repo handling & hard-coding `codeartifact`.  what would it take for us to adapt our CA config to work with the current setup here?  also, with the twine setup from `twine-login`, aren't we putting the credentials in a form that `twine` expects?  i could see perhaps needing to update the handing of `twine` in this ruleset to allow for passing through some of the config, but this replacement worries me.

Reply:
This change is to adapt CA config.

Reply:
I could see us still wanting to upload to PyPi, since we have a few open source projects that do so.